js/addToTask.js

```diff
@@ -9,14 +9,15 @@ import {
   TODAY,
   COMPLETE,
   EDIT,
-  DELET,
+  DELETE_TODO,
   CLICK_EVENT,
   HIDE,
   AFTER_BEGIN,
 } from "/js/constants.js"; //named import
+import removeTodo from "/js/deleteTask.js";
 
-export let taskList = [];
-export let id = 0;
+const taskList = [];
+let id = 0;
 let dateString;
 function addToDo(taskName, id) {
```

Comment:
separate function declaration by a line break.

```suggestion
let dateString;

function addToDo(taskName, id) {
```